src/ol_concourse/pipelines/container_images/dcind.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+from ol_concourse.lib.containers import container_build_task
+from ol_concourse.lib.models.pipeline import (
+    GetStep,
+    Identifier,
+    Input,
+    Job,
+    Pipeline,
+    PutStep,
+    Resource,
+)
+from ol_concourse.lib.resources import git_repo, github_release
+
+ol_inf_repo = git_repo(
+    name=Identifier("ol-infrastructure-repository"),
+    uri="https://github.com/mitodl/ol-infrastructure",
+    branch="md/issue_1881",
+    check_every="24h",
+    paths=["dockerfiles/dcind/"],
+)
+
+earthly_release = github_release(
+    name=Identifier("earthly-release-binary"),
+    owner="earthly",
+    repository="earthly",
+    order_by="time",
+)
+
+dcind_release_image = Resource(
+    name=Identifier("dcind-release-resource-image"),
+    type="registry-image",
+    icon="docker",
+    source={
+        "repository": "mitodl/dcind",
+        "tag": "latest",
+        "password": "((dockerhub.password))",
+        "username": "((dockerhub.username))",
+    },
+)
+
+build_task = container_build_task(
+    inputs=[Input(name=ol_inf_repo.name)],
+    build_parameters={"CONTEXT": f"{ol_inf_repo.name}/dockerfiles/dcind"},
```

Comment:
Shouldn't this also get the `BUILD_ARG` parameter for the Earthly release version?

Reply:
This was surprisingly difficult to accomplish.